AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1,4 +1,10 @@
 !
+! This section contains the list of the rules are against annoyances that can't be blocked in other files. Rules should be specific.
```

Comment:
`can't be blocked in other files` => `not falling under existing categories`. We can block literally all in this file.